src/Microsoft.Azure.Functions.ExtensionBundle/extensions.json

```diff
@@ -98,6 +98,18 @@
       "orchestrationclient"

```

Comment:
Change the target branch to `main-preview` if you need this in the preview bundle.